rocAL/rocAL/include/pipeline/tensor.h

```diff
@@ -198,10 +200,14 @@ class rocalTensorInfo {
     uint64_t _data_type_size = tensor_data_size(_data_type);
     uint64_t _data_size = 0;
     std::vector<size_t> _max_shape;  //!< stores the the width and height dimensions in the tensor
+    std::shared_ptr<std::vector<float>> _sample_rate;
+    void reallocate_tensor_sample_rate_buffers();
     void reset_tensor_roi_buffers();
     bool _is_image = false;
     bool _is_metadata = false;
     size_t _channels = 3;   //!< stores the channel dimensions in the tensor
+    std::shared_ptr<std::vector<uint32_t>> _orig_roi_width;//!< The actual image width stored in the buffer, it's always smaller than _width. It's created as a vector of pointers to integers, so that if it's passed from one tensor to another and get updated by one and observed for all.
+    std::shared_ptr<std::vector<uint32_t>> _orig_roi_height;//!< The actual image height stored in the buffer, it's always smaller than _height. It's created as a vector of pointers to integers, so that if it's passed from one tensor to another and get updated by one changes can be observed for all.
```

Comment:
```suggestion
    std::shared_ptr<std::vector<uint32_t>> _orig_roi_width;//!< The actual image width stored in the buffer, it's always smaller than _width. It's created as a vector of pointers to integers, so that if it's passed from one tensor to another and get updated by one and observed for all.
    std::shared_ptr<std::vector<uint32_t>> _orig_roi_height;//!< The actual image height stored in the buffer, it's always smaller than _height. It's created as a vector of pointers to integers, so that if it's passed from one tensor to another and get updated by one changes can be observed for all.
```

Remove these lines

Reply:
Resolved